src/Documenter.jl

```diff
@@ -194,21 +194,24 @@ function makedocs(components...; debug = false, format = HTML(),
                   html_disable_git::Union{Bool, Nothing} = nothing, # deprecated
                   html_edit_branch::Union{String, Nothing} = nothing, # deprecated
                   html_canonical::Union{String, Nothing} = nothing, # deprecated
+                  assets::Union{Vector{<:AbstractString}, Nothing} = nothing, # deprecated
+                  analytics::Union{<:AbstractString, Nothing} = nothing, # deprecated
                   kwargs...)
     # html_ keywords deprecation
     html_keywords = Dict()
     function html_warn(kw)
-        Base.depwarn("""
+        replace_with = startswith(kw, "html_") ? kw[6:end] : kw
+        @warn """
```

Comment:
Why the change from `depwarn` to `@warn`?

Reply:
Ah, good point, would be good to have your opinion on this. What I noticed is that when you are using multiple deprecated arguments you only get one warning, whereas with `@warn` you get all applicable ones. Also, the line number in the depwarn was like `Core: -1`, whereas `@warn` points to the correct place. Otherwise, it didn't seem that there was any difference to the user.

Reply:
Ok yea, lets keep it like this then.